tests/appsec/iast/sink/test_nosql_mongodb_injection.py

```diff
@@ -3,7 +3,7 @@
 # Copyright 2021 Datadog, Inc.
 
 from utils import context, missing_feature, scenarios, features
-from .._test_iast_fixtures import BaseSinkTest
+from ..utils import BaseSinkTest
```

Comment:
Unless pytest configuration is adjusted, this will lead to pytest not instrumenting asserts in `BaseSinkTest` anymore, losing "pretty asserts".

Reply:
checking

Reply:
Indeed, thanks a lot for catching this. Fixing it right now.

Reply:
-> [665567b](https://github.com/DataDog/system-tests/pull/2420/commits/665567b78def0799144aa6bf0ef525828ce8dd04)